doc/mesh_filter/launch/mesh_filter.launch

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<launch>
+
+  <node pkg="moveit_tutorials" type="filtered_depth_subscriber_node" name="filtered_depth_subscriber_node" output="screen"/>
+
+  <include file="$(find moveit_tutorials)/doc/mesh_filter/launch/test_gazebo.launch"/>
```

Comment:
includes between launch files in the tutorials do not work because the files are installed in different locations between devel and install workspaces. [I found the same issue in another tutorial yesterday](https://github.com/ros-planning/moveit_tutorials/pull/633).

Given that there is no other reference to `test_gazebo.launch`, I would just include its content here.

Reply:
Would `$(dirname)/test_gazebo.launch` work? I've not had a reason to use `dirname` before but it seems applicable here.

Reply:
> Would $(dirname)/test_gazebo.launch work?

Yes! I wonder how I managed to miss the introduction of that feature!
Last time I checked this website was before kinetic :smile: .
Thanks for pointing it out. I have to rewrite so many launch files now....

With this we can keep the separate files, but I would still propose to change the name. This is no test.
Something like `ur5_gazebo.launch` would work for me.

Also, I'm still no fan of using a different robot for this tutorial (it adds another dependency that will break at some point), but I don't know whether the panda gazebo simulation actually works at the moment.

Reply:
I chnaged the name of the launch file to `ur5_gazebo.launch`